Fixmirror

Amelia0010 opened the pull request:
https://github.com/yogstation13/Yogstation-TG/pull/2347
https://github.com/yogstation13/Yogstation-TG/pull/2346

Cruix commented:
Lawyers have read-only access to security records through their PDA, they do not have access to security records consoles because the ability to set people to arrest is purposely limited to roles with mindshield implants (ignoring the fact that the HoP can give himself the access).

matskuman5 commented:
Hm, how about we make lawyers mindshielded?

Cruix commented:
Should be a separate PR.

The console is fine, the lawyer can ask an officer to log in for him and if they trust him. I would love to see security and medical records used more than the current "antags delete their own record so they cannot be set to arrest" meta that exists now.

Partheo commented:
Very nice. It did look like a closet before. Are those wooden walls?

matskuman5 commented:
Yup @partheo

Ratvar commented:
Devil metaclub revealed, now you can fireball crew contracts at godspeed, stealthy buff alert

Zostroll commented:
The rack is a bit to industrial for the theme of the room and the lawyer has nowhere for their clients to sit. Other than that I think it looks neat!

Zostroll commented:
I just saw the seat behind the red cross...

Super3222 commented:
I see where Zostroll is going with the racks. You can change it if you want. Other than that, this looks good to go.

matskuman5 commented:
I'll fix this later

MrsTonedOne committed:
Remove the prequeue

Delaying the gc wasn't having a worthwhile notice on qdel() cost and can slow shit down

MrsTonedOne committed:
Remove prequeue defines

MrsTonedOne committed:
Remove unneeded check

SpaceManiac committed:
Refactor status displays

SpaceManiac committed:
Fix rogue var on PubbyStation

SpaceManiac committed:
Use faster loops for AI status displays

Amelia0010 committed:
FUCK
